code/script.js

```diff
@@ -2,334 +2,460 @@
+    shirtColor: 'white',
     glasses: false,
-    hat: true,
-    smoker: false,
+    moustache: false,
+    scarf: false,
+    unibrow: false
   },
   {
-    name: 'Jess',
-    img: 'images/jess.svg',
-    hairColor: 'black',
-    eyeColor: 'blue',
-    glasses: true,
-    hat: false,
-    smoker: false,
+    name: 'Lee',
+    img: 'images2/Lee.jpg',
+    hatColor: 'black',
+    shirtColor: 'white',
+    glasses: false,
+    moustache: false,
+    scarf: false,
+    unibrow: false
   },
   {
-    name: 'Jocelyn',
-    img: 'images/jocelyn.svg',
-    hairColor: 'black',
-    eyeColor: 'brown',
-    glasses: true,
-    hat: false,
-    smoker: false,
+    name: 'Logan',
+    img: 'images2/Logan.jpg',
+    hatColor: 'captains',
+    shirtColor: 'checkered',
+    glasses: false,
+    moustache: true,
+    scarf: false,
+    unibrow: false
   },
   {
-    name: 'Jon',
-    img: 'images/jon.svg',
-    hairColor: 'brown',
-    eyeColor: 'green',
+    name: 'Lucie',
+    img: 'images2/Lucie.jpg',
+    hatColor: 'beige',
+    shirtColor: 'white',
     glasses: true,
-    hat: false,
-    smoker: false,
+    moustache: false,
+    scarf: false,
+    unibrow: false
   },
   {
-    name: 'Jordan',
-    img: 'images/jordan.svg',
-    hairColor: 'yellow',
-    eyeColor: 'hidden',
+    name: 'Lennon',
+    img: 'images2/Lennon.jpg',
+    hatColor: 'yellow',
+    shirtColor: 'checkered',
     glasses: true,
-    hat: true,
-    smoker: false,
+    moustache: true,
+    scarf: false,
+    unibrow: true
   },
   {
-    name: 'Josephine',
-    img: 'images/josephine.svg',
-    hairColor: 'grey',
-    eyeColor: 'brown',
+    name: 'Lynn',
+    img: 'images2/Lynn.jpg',
+    hatColor: 'dark blue',
+    shirtColor: 'yellow',
     glasses: false,
-    hat: false,
-    smoker: false,
+    moustache: false,
+    scarf: false,
+    unibrow: false
   },
   {
-    name: 'Josh',
-    img: 'images/josh.svg',
-    hairColor: 'yellow',
-    eyeColor: 'green',
+    name: 'Larry',
+    img: 'images2/Larry.jpg',
+    hatColor: 'pink',
+    shirtColor: 'black',
     glasses: false,
-    hat: false,
-    smoker: false,
+    moustache: true,
+    scarf: false,
+    unibrow: false 
   },
-  {
-    name: 'Jude',
-    img: 'images/jude.svg',
-    hairColor: 'black',
-    eyeColor: 'green',
-    glasses: false,
-    hat: false,
-    smoker: false,
+  { 
+    name: 'Lindsey',
+    img: 'images2/Lindsey.jpg',
+    hatColor: 'green',
+    shirtColor: 'white',
+    glasses: true,
+    moustache: false,
+    scarf: false,
+    unibrow: true
   },
   {
-    name: 'Julie',
-    img: 'images/julie.svg',
-    hairColor: 'black',
-    eyeColor: 'brown',
+    name: 'Lenwood',
+    img: 'images2/Lenwood.jpg',
+    hatColor: 'black',
+    shirtColor: 'yellow',
     glasses: true,
-    hat: true,
-    smoker: false,
+    moustache: true,
+    scarf: false,
+    unibrow: false
   },
 ]
 
 // Global variables
-let secret, currentQuestion, charactersInPlay
+
+let currentQuestion 
+
+let secret  
```

Comment:
Can we still reach charactersInPlay when it´s no longer a global variable?
Do we want the blank lines between the let?

Reply:
It works without setting characters.InPlay to a global variable! I'm not sure why...  i't declared in start()

Reply:
The true art is to not use global variables at all. You will learn it soon 👨🏻‍🏫